postman/threads-api.postman_collection.json

```diff
@@ -2465,43 +3040,46 @@
 		},
 		{
 			"key": "scope",
-			"value": "threads_basic,threads_content_publish,threads_manage_insights,threads_manage_replies,threads_read_replies"
+			"value": "threads_basic,threads_content_publish,threads_manage_insights,threads_manage_replies,threads_read_replies,threads_keyword_search,threads_manage_mentions"
 		},
 		{
 			"key": "media_type",
-			"value": "TEXT_POST, IMAGE, VIDEO, CAROUSEL"
+			"value": "TEXT, IMAGE, VIDEO, CAROUSEL"
 		},
 		{
 			"key": "reply_control",
 			"value": "everyone,accounts_you_follow,mentioned_only"
 		},
 		{
 			"key": "metrics_post",
-			"value": "views,likes,replies,reposts,quotes,reposts"
+			"value": "views,likes,replies,reposts,quotes,shares",
+			"type": "string"
 		},
 		{
 			"key": "metrics_account",
-			"value": "views,likes,replies,reposts,quotes,reposts, followers_count, follower_demographics"
+			"value": "views,likes,replies,reposts,quotes,followers_count,follower_demographics",
+			"type": "string"
 		},
 		{
 			"key": "follower_demographics_breakdown",
-			"value": "country,city,age,gender"
+			"value": "country,city,age,gender",
+			"type": "string"
 		},
 		{
 			"key": "fields_threads",
 			"value": "id,media_product_type,media_type,media_url,permalink,owner,username,text,timestamp,shortcode,thumbnail_url,children,is_quote_post,has_replies,alt_text,link_attachment_url"
 		},
 		{
 			"key": "fields_replies",
-			"value": "id,text,timestamp,media_product_type,media_type,media_url,permalink,shortcode,thumbnail_url,username,children,is_quote_post,has_replies,is_reply,is_reply_owned_by_me,root_post,replied_to,hide_status,reply_audience"
+			"value": "id,text,timestamp,media_product_type,media_type,media_url,permalink,shortcode,thumbnail_url,username,children,is_quote_post,alt_text,link_attachment_url,has_replies,is_reply,is_reply_owned_by_me,root_post,replied_to,hide_status,reply_audience"
```

Comment:
Can we add `quoted_post` and `reposted_post` to this list of fields?

Reply:
Done